pkg/controller/telemetry/accesslog.go

```diff
@@ -36,7 +36,9 @@ type logInfo struct {
 
 func OutputAccesslog(data requestMetric, accesslog logInfo) {
 	logStr := buildAccesslog(data, accesslog)
-	fmt.Println("accesslog:", logStr)
+	if accesslog.destinationService != "" || data.dstPort == uint16(15019) {
+		fmt.Println("accesslog:", logStr)
```

Comment:
I am confusing for other cases if you donot print accesslog, why not skip buildAccesslog too